src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -739,6 +776,27 @@ private static void addColonyByWorld(Colony colony)
     }
 
     /**
+     * Set the server UUID.
+     *
+     * @param uuid the universal unique id
+     */
+    public static void setServerUUID(final UUID uuid)
+    {
+        serverUUID = uuid;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'serverUUID' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)